src/pages/base/homepage/components/TotalAtRiskAssetSection.js

```diff
@@ -0,0 +1,56 @@
+import {
+  faChartBar,
+  faChartArea,
+  faInfoCircle,
+} from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import React, { useState } from "react";
+import { Col, Row } from "reactstrap";
+import IconTabs from "../../../../components/Tabs/IconTabs.js";
+import { withErrorBoundary } from "../../../../hoc.js";
+import TotalAtRiskAssetChart from "./TotalAtRiskAssetChart.js";
+
+function TotalAtRiskAssetSection(props) {
+  const [modalOpen, setModalOpen] = useState(false);
+  const toggleModalOpen = () => setModalOpen(!modalOpen);
+
+  let description;
+
+  description =
```

Comment:
You don't need to create a new variable for this. Just put the text directly inside the <p> where you currently have a `{description}`.
Since you're not modifying it anywhere just put it straight in (aka. hardcode it) :)